packages/stable/src/api/files/filesApi.ts

```diff
@@ -233,6 +248,52 @@ export class FilesAPI extends BaseResourceAPI<FileInfo> {
     );
     return this.addToMapAndReturn(response.data.items, response);
   }
+  /**
+   * [Init a multipart file upload](https://api-docs.cognite.com/20230101/tag/Files/operation/initMultiPartUpload)
+   *
+   * ```js
+   * const numberOfParts = 5;
+   * // automatic upload:
+   * const multiPartUploadApi = await client.files.multipartUploadSession({name: 'examplefile.jpg', mimeType: 'image/jpeg'}, numberOfParts);
+   * ```
+   */
+  public async multipartUploadSession(
+    fileInfo: ExternalFileInfo,
+    parts: number,
+    overwrite = false
+  ) {
+    const response = await this.getMultipartUploadSession(
+      fileInfo,
+      parts,
+      overwrite
+    );
+    const multipartUploadSession = new FilesMultipartUploadSessionAPI(
+      this._baseUrl,
+      this._client,
+      this._map,
+      response.data
+    );
+    return this.addToMapAndReturn(multipartUploadSession, response);
+  }
+  private async getMultipartUploadSession(
+    fileInfo: ExternalFileInfo,
+    parts: number,
+    overwrite = false
+  ) {
+    if (
+      parts < this.limits.minimumNumberOfParts ||
+      parts > this.limits.maxNumberOfParts
+    ) {
+      throw Error('parts must be greater than 0 and less than 250');
```

Comment:
Suggestion: don't hard code dynamic values in the string. If we update limits, then the error message should reflect that.
```suggestion
      throw Error(`parts must be in range ${this.limits.minimumNumberOfParts} <= parts <= and less than ${this.limits.maxNumberOfParts}`);
```